tools/scoring/score/dimensions/standard_field_naming.py

```diff
@@ -31,17 +31,34 @@ class StandardFieldNaming(Dimension):
   def __init__(self, *, translations: TranslationsDict):
     super().__init__(translations=translations)
 
+    # Combine translations for all devices within the dictionary
+    solution_condensed = [
+        matched_translations[SOLUTION]
+        for matched_translations in translations.values()
+        if matched_translations[SOLUTION]
+    ]
+    # Account for empty list
+    solution_translations = solution_condensed and solution_condensed[0]
+
+    proposed_condensed = [
+        matched_translations[PROPOSED]
+        for matched_translations in translations.values()
+        if matched_translations[PROPOSED]
+    ]
```

Comment:
same here, create a helper function, it will reduce the code and make it more readable :)